docs/docs/tutorials/multi_agent/agent_supervisor.ipynb

```diff
@@ -12,15 +12,13 @@
    "source": [
     "# Multi-agent supervisor\n",
     "\n",
-    "The [previous example](../multi-agent-collaboration) routed messages automatically based on the output of the initial researcher agent.\n",
+    "In this tutorial, you will build a [multi-agent supervisor](../../../concepts/multi_agent#supervisor) system.\n",
     "\n",
-    "We can also choose to use an [LLM to orchestrate](https://langchain-ai.github.io/langgraph/concepts/multi_agent/#supervisor) the different agents.\n",
-    "\n",
-    "Below, we will create an agent group, with an agent supervisor to help delegate tasks.\n",
+    "**Supervisor** is an architecture where **specialized** agents are coordinated by a central **supervisor agent**. The supervisor agent controls all communication flow and task delegation, making decisions about which agent to invoke based on the current context and task requirements.\n",
     "\n",
     "![diagram](attachment:8ee0a8ce-f0a8-4019-b5bf-b20933e40956.png)\n",
     "\n",
-    "To simplify the code in each agent node, we will use LangGraph's prebuilt [create_react_agent](https://langchain-ai.github.io/langgraph/reference/prebuilt/#langgraph.prebuilt.chat_agent_executor.create_react_agent). This and other \"advanced agent\" notebooks are designed to show how you can implement certain design patterns in LangGraph. If the pattern suits your needs, we recommend combining it with some of the other fundamental patterns described elsewhere in the docs for best performance.\n",
+    "By the end of tutorial you will create an agent team with a supervisor and two agents — a research agent and a math expert.\n",
     "\n",
     "## Setup\n",
```

Comment:
```suggestion
    "## 1. Setup\n",
```
I think the headings should be numbered to clearly indicate that it's a step-by-step tutorial

Reply:
added numbers, but lmk if setup should be included as well